### PR TITLE
Tab bar for desktop/desktop 974

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

This adds the tab bar to the main app.
 

screen shot 2016-05-05 at 6 30 58 pm
 
screen shot 2016-05-05 at 6 41 30 pm

screen shot 2016-05-05 at 6 42 02 pm

### Todo

* Figure out why the dumb sheet makes the desktop app fail without a filter `''`

* Add correct icon for search

* Rebuild fonts (While rebasing before pushing I got a merge conflict with the kb.ttf)

* Small cleanups

### CODE REVIEW

**Comment by chrisnojima** [comment]
Let's walk through the font stuff

File: shared/images/icons/README.md [hunk 1]
```diff
@@ -20,8 +20,24 @@ Leave the rest as the defaults
...
 
-Update the fonts on the React native side!
+## Fixing Vertical Metrics
 
+Go [here](https://www.fontsquirrel.com/tools/webfont-generator)
+
+1. choose expert
+1. upload the font
+1. only select trueType in the formats
+1. remove teh ‘-webfont’ as the suffix
```
Comment (chrisnojima) [inline shared/images/icons/README.md]: teh -> the

File: shared/nav.desktop.js [hunk 2]
```diff
@@ -166,62 +170,37 @@ class Nav extends Component {
       return <div>Coming soon!</div>
     }
 
+    const tabContent = mapValues(tabs, ({module}, tab) => (activeTab === tab && this._renderContent(tab, module)))
+
+    console.log('tabc', tabContent)
```
Comment (chrisnojima) [inline shared/nav.desktop.js]: 🔥

**Comment by chrisnojima** [comment]
👍

File: shared/common-adapters/dumb.desktop.js [hunk 3]
```diff
@@ -1,27 +1,77 @@
...
+    'Disabled - unchecked': {
+      label: 'Disabled - unchecked',
+      onCheck,
+      disabled: true,
+      checked: false
     }
   }
 }
+
+const IconButton = ({selected, icon, badgeNumber}: any) => <TabBarButton source={{type: 'icon', icon}} selected={selected} badgeNumber={badgeNumber} style={{height: 40}}/>
```
Comment (chromakode) [inline shared/common-adapters/dumb.desktop.js]: Missing space before the `/>` (at the end of line) here?

File: shared/tab-bar/index.render.desktop.js [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  return labels[t]
+}
+
+export default class Render extends Component<void, Props, void> {
+  _renderSearch (onPress: () => void, searchActive: boolean) {
+    const backgroundColor = searchActive ? globalColors.orange : globalColors.darkBlue
+    const button = (
+      <Box style={{...globalStyles.flexBoxColumn, padding: 24}}>
+        <Box style={{...stylesSearchButton, backgroundColor}}>
+          <Icon type='fa-custom-main-nav-people' style={{color: globalColors.white, fontSize: 24}}/>
```
Comment (chromakode) [inline shared/tab-bar/index.render.desktop.js]: also no space before `/>` here

Comment (MarcoPolo) [inline shared/tab-bar/index.render.desktop.js]: I thought we had an eslint rule for this? @chrisnojima

**Comment by chromakode** [comment]
:+1:

File: shared/tab-bar/index.render.desktop.js (hunk 4 above)
Comment (chrisnojima) [inline shared/tab-bar/index.render.desktop.js]: we do, dunno why your ide isn't complaining. what does the commandline say

Comment (MarcoPolo) [inline shared/tab-bar/index.render.desktop.js]: no errors, and pre-commit hooks pass

Comment (chromakode) [inline shared/tab-bar/index.render.desktop.js]: I'm seeing this from running eslint elsewhere in the codebase, it looks like:

```
   88:24   error  A space is required before closing bracket  react/jsx-space-before-closing
```

Comment (MarcoPolo) [inline shared/tab-bar/index.render.desktop.js]: yeah my eslint was out of date. It's fixed now

**Pull request merged**